src/DssSpell.sol

```diff
@@ -49,100 +51,33 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
-        DssExecLib.deauthorize(MCD_CAT, MCD_END_OLD);
-
-        require(MCD_DOG == EndAbstract(MCD_END_OLD).dog(), "sanity-error-dog");
-        DssExecLib.setContract(MCD_END, "dog", MCD_DOG);
-        DssExecLib.authorize(MCD_DOG, MCD_END);
-        DssExecLib.deauthorize(MCD_DOG, MCD_END_OLD);
-
-        require(MCD_SPOT == EndAbstract(MCD_END_OLD).spot(), "sanity-error-spot");
-        DssExecLib.setContract(MCD_END, "spot", MCD_SPOT);
-        DssExecLib.authorize(MCD_SPOT, MCD_END);
-        DssExecLib.deauthorize(MCD_SPOT, MCD_END_OLD);
-
-        require(MCD_POT == EndAbstract(MCD_END_OLD).pot(), "sanity-error-pot");
-        DssExecLib.setContract(MCD_END, "pot", MCD_POT);
-        DssExecLib.authorize(MCD_POT, MCD_END);
-        DssExecLib.deauthorize(MCD_POT, MCD_END_OLD);
-
-        DssExecLib.setContract(MCD_ESM, "end", MCD_END);
-        DssExecLib.authorize(MCD_END, MCD_ESM);
-        DssExecLib.deauthorize(MCD_END_OLD, MCD_ESM);
-
-        bytes32[] memory ilks = IlkRegistryAbstract(ILK_REGISTRY).list();
-        for (uint256 i = 0; i < ilks.length; i++) {
-            bytes32 ilk = ilks[i];
-
-            address xlip = IlkRegistryAbstract(ILK_REGISTRY).xlip(ilk);
-            if (IlkRegistryAbstract(ILK_REGISTRY).class(ilks[i]) < 3) {
-                DssExecLib.deauthorize(xlip, MCD_END_OLD);
-                DssExecLib.authorize(xlip, MCD_END);
-            }
-
-            try DssExecLib.removeReaderFromWhitelist(IlkRegistryAbstract(ILK_REGISTRY).pip(ilk), MCD_END_OLD) {} catch {}
-            try DssExecLib.addReaderToWhitelist(IlkRegistryAbstract(ILK_REGISTRY).pip(ilk), MCD_END) {} catch {}
-        }
-
-        // --------------------------- New DssFlash ----------------------------
-
-        DssExecLib.authorize(MCD_VAT, MCD_FLASH);
-        uint256 newMax = FlashAbstract(MCD_FLASH_LEGACY).max() / 2;
-        DssExecLib.setValue(MCD_FLASH, "max", newMax);
-        DssExecLib.setValue(MCD_FLASH_LEGACY, "max", newMax);
-
-        // ------------------------ Transfer MKR to SES ------------------------
-        DSTokenAbstract(MCD_GOV).transfer(SES_WALLET, 541.47 ether);
-
-        // ----------------------------- Chainlog ------------------------------
-
-        DssExecLib.setChangelogAddress("MCD_END", MCD_END);
-        DssExecLib.setChangelogAddress("MCD_CURE", MCD_CURE);
-        DssExecLib.setChangelogAddress("MCD_FLASH", MCD_FLASH);
-        DssExecLib.setChangelogAddress("MCD_FLASH_LEGACY", MCD_FLASH_LEGACY);
-        DssExecLib.setChangelogVersion("1.13.0");
+        // ---------------------------- Lid for Flap ---------------------------
+        DssExecLib.setValue(MCD_FLAP, "lid", 30_000 * RAD);
+
+        // ------------------------------ ESM Min ------------------------------
+        DssExecLib.setValue(MCD_ESM, "min", 150_000 * WAD);
+
+        // ---------------------------- Transfer MKR ---------------------------
+        DSTokenAbstract(MCD_GOV).transfer(SIDESTREAM_WALLET, 243.7953 ether);
```

Comment:
For the record, I hate the `ether` keyword being used for anything but Ether, but it works and we can leave it. I just needed a moment to vent here.

Reply:
Patrick and I were also like 'wtf is this', then looked it up and went about our days :)

Reply:
haha yeah tbh, I have to take a second look each time, but from https://github.com/makerdao/spells-mainnet/blob/master/archive/2022-05-18-DssSpell/DssSpell.sol#L137 I assumed it was an optimization/standard since i came back... now i want to change it but sadly not worth the re-reviews